apps/root/src/pages/transfer/components/recipient-address/components/contacts-button/index.tsx

```diff
@@ -28,8 +27,8 @@ const StyledContactsButton = styled(ForegroundPaper).attrs({ elevation: 0 })`
 
 const ContactsButton = ({ onClick }: ContactsButtonProps) => (
   <StyledContactsButton onClick={onClick}>
-    <Profile2UsersIcon />
-    <Typography variant="bodyExtraExtraSmallBold" color={({ palette }) => colors[palette.mode].accentPrimary}>
+    <Profile2UsersIcon sx={({ palette }) => ({ color: colors[palette.mode].accentPrimary })} />
+    <Typography variant="bodyExtraExtraSmallBold" color="primary">
```

Comment:
hold up there cowboy, lets not trust the color property of the typography component